pkg/storaged/overview.js

```diff
@@ -417,6 +418,21 @@ define([
                                 }
                               }
                           ],
+                          ChangeHandler: function (dialog) {
+                              // hide the chunk size selector for RAID 1, as the chunk size setting doesn't make sense
```

Comment:
The value of a hiden field is guaranteed to be `undefined` in the action function, so that we can use 0 in that case to satisfy storaged, I think.